ucsmsdk_samples/firmware/ucsfirmware.py

```diff
@@ -52,6 +52,20 @@ def firmware_available(username, password, mdf_id_list=None, proxy=None):
     image_names = [image.image_name for image in images]
     return sorted(image_names)
 
+def return_download_state(handle, file_name):
+    '''
+    Returns Transfer Status of  FirmwareDownloader object for a single download. 
+    
+    Args:
+        handle (UcsHandle)
+        file_name (string): firmware image name
+
+    Example:
+        return_download_status(handle, 'ucs-k9-bundle-infra.2.2.5b.A.bin')
+    '''
+    download_state = (h.query_dn(dn='sys/fw-catalogue/dnld-%s' % file_name)).transfer_state
```

Comment:
`h` -> `handle`.
Check and raise exception `UcsOperationError` if `handle.query_dn` returns None.